computerVision:
  lightSignalDetection:
    greenRoi:
      xmin: 0.30
      xmax: 0.38
      ymin: 0.53
      ymax: 0.62
    yellowRoi:
      xmin: 0.30
      xmax: 0.38
      ymin: 0.45
      ymax: 0.53
    redRoi:
      xmin: 0.30
      xmax: 0.38
      ymin: 0.35
      ymax: 0.44
